### ISSUE TITLE
Template is missing

### ISSUE THREAD

**Issue by whitered** [bradphelan/jasminerice#2]
I've followed all the steps from the readme and got this error on the localhost:3000/jasmine

```
Template is missing

Missing template jasminerice/spec/index, jasminerice/application/index with {:handlers=>[:erb, :builder, :slim], :formats=>[:html], :locale=>[:ru, :ru]}. Searched in: * "/home/whitered/projects/p27/app/views" * "/home/whitered/.rvm/gems/ruby-1.9.2-p180/gems/jasminerice-0.0.6/app/views" * "/home/whitered/.rvm/gems/ruby-1.9.2-p180/gems/devise-1.4.2/app/views"
```

**Comment by magicienap** [comment]
I added `gem 'haml'` in my Gemfile, because jasminerice uses haml to generate template and that worked!

**Comment by bradphelan** [comment]
Which version are you using. In the gemspec

https://github.com/bradphelan/jasminerice/blob/master/jasminerice.gemspec#L11

I include the haml dependency. Is this still a problem if you use the latest version?

**Comment by magicienap** [comment]
I try it (with the new version) and that doesn't seem to work for me. HAML is installed by Bundler, but if I don't add `gem 'haml'` in my Gemfile, I get the same error.

**Comment by bradphelan** [comment]
Then I'm missing something about the way gemspec and bundler is meant
to interact. I'll think about it.

**Comment by whitered** [comment]
I use the latest version that has the dependency defined and it does not work. Adding `gem 'haml'` in Gemfile solved the issue

**Comment by bradphelan** [comment]
I added a stack overflow question about this issue.

http://stackoverflow.com/questions/6826235/gemspecs-and-bundler-in-rails-plugin

**Comment by kbackowski** [comment]
Hi, 

Firstly many thanks for this gem it's just what i was looking for to test my rails backbone application :)

However i was wondering if you have considered removing haml from dependencies. You have only one view which just includes needed javascripts and therefore using haml is pointless. We're writing rails 3.1 application but htmls are created by our UI team which is not familiar with haml. I would like to use your gem without forking ;)

**Comment by bradphelan** [comment]
Why do you have to fork? Haml is used internally and if you ignore it you
won't care that is there. You are not forced to use Haml in any other file
at all.
On Aug 25, 2011 8:39 PM, "keybi" <
reply@reply.github.com>
wrote:

**Comment by kbackowski** [comment]
But i need to have haml in my dependencies of my application ...

I'm just suggesting that using haml in a gem just for 10 lines of html code is a bit overuse.

**Comment by bradphelan** [comment]
I would suggest that  inserting 40 lines of erb to replace 10 lines of Haml
because somebody doesn't like one line in the gemfile is a bit extreme?
Anyway in theory you shouldn't require the Haml in the gemfile as its in the
gemspec dependencies but that doesn't work for some reason. If you could
investigate what is wrong then I suspect your issue will be solved.
On Aug 25, 2011 9:17 PM, "keybi" <
reply@reply.github.com>
wrote:

**Comment by bradphelan** [comment]
There is a fix on release 0.0.7 which fixes the haml require bug. Now you don't have to require it in your Gemfile.
